### ISSUE TITLE
[Ltac2] No (obvious) support for goal selectors

### ISSUE THREAD

**Issue by MSoegtropIMC** [coq/coq#10386]
In Ltac2 I can't find any support for goal selectors, that is what is described for Ltac1 here:

<https://coq.github.io/doc/master/refman/proof-engine/ltac.html?highlight=all#goal-selectors>

This means that I either have to solve a complete goal, say induction with all its cases, with a single tactic or to give at least one tactic for each goal.

The proof style I prefer is to first give a few lines to describe the basic proof idea (induction, destruction, some eapply, ...) and then use automation on all goals resulting from this. I could chain both with `;` but I like to see the goals resulting from step 1. So most of my proofs like like this:

```
<some rough proof sketch>
all: <some automation to sort out the details>
```

The `all:` is lacking from Ltac2, so I don't know how to do this with Ltac2.

Btw.: I tried `Control.focus`, but as far as I can tell with this I can only restrict the focus, but not extend it.

**Comment by JasonGross** [comment]
You can probably get around this for now with `Set Default Goal Selector all `

**Comment by MSoegtropIMC** [comment]
@JasonGross : thanks, a neat trick! I removed the priority high - with this trick I can live for the time being.

P.S.: (for others needing this) the syntax is `Set Default Goal Selector "all".`

**Comment by ppedrot** [comment]
Fixed by #15378.
